Dockerfile

```diff
@@ -0,0 +1,122 @@
+FROM alpine:3.7 as builder
+
+RUN apk add --no-cache \
+     ca-certificates \
+     autoconf \
+     automake \
+     build-base \
+     libressl \
+     libtool \
+     gmp-dev \
+     python \
+     python-dev \
+     python3 \
+     sqlite-dev \
+     wget \
+     git \
+     file \
+     gnupg \
+     swig
```

Comment:
This should also install zlib (see https://github.com/ElementsProject/lightning/commit/083a2cee0744e93c9f43ab553466b0c6fe01867f#diff-04c6e90faac2675aa89e2176d2eec7d8)